src/tests/streammaster_test.cxx

```diff
@@ -141,6 +141,37 @@ TEST(Streammaster, StartTime) {
   sm.start_time(ESSTimeStamp(10));
 }
 
+// TEST(Streammaster, Statistics) {
```

Comment:
I guess the commented out implementation and test are here because they are part of functionality that you intend to include in a future PR? If so, I think it would be better to remove the commented code from here, make a branch from this one and add it, uncommented, to the new branch.